src/components/swap/PendingModalContent/index.tsx

```diff
@@ -146,70 +145,89 @@ function getContent(args: ContentArgs): PendingModalStep {
     chainId,
   } = args
 
-  switch (step) {
-    case ConfirmModalState.WRAPPING:
+  const pendingConfirmationContent = useMemo(() => {
```

Comment:
could you move this block to its own function such as `getPendingConfirmationContent` to help maintain the readability of the hook